### ISSUE TITLE
Feature request: Stop node from ANY directory

### ISSUE THREAD

**Issue by mradkov** [aeternity/aepp-aeproject-js#247]
People using `aeproject` should be able to stop the running instances of it from **ANY** directory.

Steps to reproduce: 
- Create an aeproject
- Spin up a node via `aeproject node`
- Go back, and create another project
- Try to use `aeproject node --stop` in second project's directory to stop the docker instances.

```
➜  waellet-contracts git:(master) ✗ aeproject node --stop
===== Node is not running! =====
➜  waellet-contracts git:(master) ✗ aeproject node

===== Port [3001] is already allocated! Process will be terminated! =====
Cannot start AE node, port is already allocated!
➜  waellet-contracts git:(master) ✗ docker ps -a
CONTAINER ID        IMAGE                            COMMAND                  CREATED             STATUS                      PORTS                                      NAMES
74020d07ad84        nginx:1.13.8                     "nginx -g 'daemon of…"   4 days ago          Up 4 days                   80/tcp, 0.0.0.0:3001-3003->3001-3003/tcp   example_todo_proxy_1
534c92f08f92        aeternity/aesophia_http:v3.2.0   "/docker-entrypoint.…"   4 days ago          Up 4 days (healthy)         0.0.0.0:3080->3080/tcp                     example_todo_compiler_1
ad245be4fd18        aeternity/aeternity:v4.1.0       "/docker-entrypoint.…"   4 days ago          Up 4 days (healthy)         3013-3015/tcp, 3113/tcp                    example_todo_node1_1
3e7272c06878        aeternity/aeternity:v4.1.0       "/docker-entrypoint.…"   4 days ago          Up 4 days (healthy)         3013-3015/tcp, 3113/tcp                    example_todo_node3_1
cb488664ec9d        aeternity/aeternity:v4.1.0       "/docker-entrypoint.…"   4 days ago          Up 4 days (healthy)         3013-3015/tcp, 3113/tcp                    example_todo_node2_1
3246727c2e2b        aeternity/aesophia_http:v3.2.0   "/docker-entrypoint.…"   6 weeks ago         Exited (255) 3 weeks ago    0.0.0.0:3080->3080/tcp                     addressbook-aepp-aeternity_compiler_1
ce121ee85a39        aeternity/aeternity:v3.2.0       "/docker-entrypoint.…"   6 weeks ago         Exited (255) 3 weeks ago    3013-3015/tcp, 3113/tcp                    addressbook-aepp-aeternity_node1_1
1fc414c88fde        aeternity/aeternity:v3.2.0       "/docker-entrypoint.…"   6 weeks ago         Exited (255) 3 weeks ago    3013-3015/tcp, 3113/tcp                    addressbook-aepp-aeternity_node2_1
a2296f94d779        aeternity/aeternity:v3.2.0       "/docker-entrypoint.…"   6 weeks ago         Exited (255) 3 weeks ago    3013-3015/tcp, 3113/tcp                    addressbook-aepp-aeternity_node3_1
19396ef3e47e        nginx:1.13.8                     "nginx -g 'daemon of…"   6 weeks ago         Exited (255) 3 weeks ago    80/tcp, 0.0.0.0:3001-3003->3001-3003/tcp   addressbook-aepp-aeternity_proxy_1
8840edef2149        cypress/browsers                 "node"                   4 months ago        Exited (255) 3 months ago                                              wizardly_clarke
➜  waellet-contracts git:(master) ✗
```

**Comment by maptuhec** [comment]
@mradkov hey this issue is already in development and it will be added in the next release probably. Thanks for the feedback. The reason why it is not currently working is because of some limitations of the docker-compose. 

Stay tuned for the next updated version.

Martin

**Comment by HristiyanG** [comment]
Hey @mradkov, 
Your issue has been resolved in #273 
This has been added in the latest release of aeproject @2.0.1

Hope you enjoy it.

Hristiyan

**Comment by mradkov** [comment]
Thanks!
